rb/lib/selenium/webdriver/common/options.rb

```diff
@@ -91,6 +91,14 @@ def add_option(name, value = nil)
         @options[name] = value
       end
 
+      def enable_bidi!
```

Comment:
Can we allow passing it to the constructor as well?

```ruby
Selenium::WebDriver::Options.chrome(bidi: true)
```

Reply:
I have been looking into this today to see if we can merge this branch, I hope that is okey @titusfortner 

But I'm wondering, we can already pass web_socket_url, which is the opt-in for using the BiDi protocol, so is this something we actually want like an alias for web_socket_url?

https://w3c.github.io/webdriver-bidi/#websocket-url

Reply:
I added support for 

Selenium::WebDriver::Options.chrome(bidi: true)